packages/web-app-files/l10n/translations.json

```diff
@@ -1 +1 @@
arer":"Volver compartir","Resolving private link…":"Resolvendo a ligazón privada…","Resource not found":"Non se atopou o recurso","Resource with name %{name} already exists":"Xa existe o frecurso nome %{name}","Restoration of %{file} failed":"Produciuse un fallo ao restaurar %{file}","Restore":"Restaurar","Role":"Rol","Save":"Gardar","Save Share":"Gardar a compartición","Saving":"Gardando","Saving Public Link":"Gardando ligazón pública","Saving Share":"Gardando compartición","Send a copy to myself":"Envíame unha copia","Set detailed permissions":"Estabelecer os permisos detallados","Share":"Compartir","Shared by %{resharer}":"Compartido por %{resharer}","Shared by:":"Compartido por:","Shared with":"Compartido con","Shared with me":"Compartido comigo","Shared with others":"Compartido con outros","Show Details":"Amosar os detalles","Size":"Tamaño","The name \"%{name}\" is already taken":"O nome «%{name}» xa está ocupado","The name cannot be empty":"O nome non pode estar baleiro","The name cannot be equal to \"..\"":"O nome non pode ser igual a «..»","The name cannot be equal to \".\"":"O nome non pode ser igual a «.»","The name cannot contain \"/\"":"O nome non pode conter unha «/»","The name cannot end with whitespace":"O nome non pode rematar cun espazo en branco","The private link has been copied to your clipboard.":"A ligazón privada foi copiada no portapapeis.","There are no resources in this folder.":"Non hai recursos neste cartafol.","this":"este","This resource is password-protected.":"Este recurso está protexido por contrasinal.","title":"título","Unknown Role":"Rol descoñecido","Unknown type":"Tipo descoñecido","Upload complete":"Envío completado","Upload failed":"Produciuse un fallo no envío","Upload File":"Enviar ficheiro","Upload Folder":"Enviar cartafol","Upload of a folder is not supported in Internet Explorer.":"O envío dun cartafol non é compatíbel co Internet Explorer.","Upload, edit, delete, download and preview":"Enviar, editar, eliminar, descargar e visionar","Upload, edit, delete, download, preview and share":"Enviar, editar, eliminar, descargar visionar e compartir","Uploader":"Xestor de envíos","Uploading \"%{ fileName }\"":"Enviando «%{ fileName }»","Uploading %{ count } item":["Enviando %{ count } elemento","Enviando %{ count } elementos"],"Uploading file \"%{fileName}\"":"Enviando o ficheiro «%{fileName}»","User":"Usuario","Versions":"Versións","Via %{folderName}":"Mediante %{folderName}","Viewer":"Visor","We went looking everywhere, but were unable to find the selected resource.":"Buscamos por todas partes, mais non puidemos atopar o recurso seleccionado.","You don't have permission to share this file.":"Non ten permiso para compartir este ficheiro.","You don't have permission to share this folder.":"Non ten permiso para compartir este cartafol.","You have no permission to upload!":"Non ten permiso para facer envíos!","You have not enough space left to upload!":"Non dispón de espazo abondo para facer un envío!"},"it":{}}
```

Comment:
Not sure what happened here (except for the editor removing the empty line at the end of the file). Could you omit these changes from your PR? Updating translations happens automatically